datafusion/common/src/config.rs

```diff
@@ -268,6 +268,17 @@ config_namespace! {
         /// Defaults to the number of CPU cores on the system
         pub planning_concurrency: usize, default = num_cpus::get()
 
+        /// When set to true, skips verifying that the schema produced by
+        /// planning the input of `LogicalPlan::Aggregate` exactly matches the
+        /// schema of the input plan.
+        ///
```

Comment:
wondering if blank lines can break the formatting and tests eventually failed

Reply:
I think the reason the tests failed before is that I forgot to update the `information_schema.slt` test